reorder_python_imports.py

```diff
@@ -520,29 +520,39 @@ def _report_diff(contents, new_contents, filename):
     print(diff, end='')
 
 
-FUTURE_IMPORTS = (
-    ('py22', ('nested_scopes',)),
-    ('py23', ('generators',)),
-    ('py26', ('with_statement',)),
+VERSION_IMPORTS = (
+    ('py22', ('nested_scopes',), ''),
+    ('py23', ('generators',), ''),
+    ('py26', ('with_statement',), ''),
     (
         'py3',
         ('division', 'absolute_import', 'print_function', 'unicode_literals'),
+        '',
     ),
-    ('py37', ('generator_stop',)),
-)  # type: Tuple[Tuple[str, Tuple[str, ...]], ...]
+    (
+        'py33',
+        ('division', 'absolute_import', 'print_function', 'unicode_literals'),
+        '. Rewrites mock imports to use unittest.mock',
```

Comment:
this seems slightly too clever. maybe we just dump the complicated working and say "the pyXX option removes / replaces imports which are unnecessary in that version"

Reply:
Should I remove the `__future__` enumeration too?